ocs_ci/cleanup/vsphere/cleanup.py

```diff
@@ -0,0 +1,215 @@
+"""
```

Comment:
thinking maybe we can create a class for each platform using factory design, the same as we have for deployment

Reply:
I will take up this later as a separate PR